### PR TITLE
#27500 PHPUnit 9 

### PR DESCRIPTION
**Pull request by lbajsarowicz**
### Description (*)
Prepare a few Unit Tests to work native with PHPUnit 9

### Fixed Issues (if relevant)

1. magento/inventory#<issue_number>: Issue title

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by sidolov — DISMISSED** [review]

File: InventoryInStorePickupMultishipping/Test/Unit/Controller/Checkout/PluginTest.php [hunk 1]
```diff
@@ -40,7 +40,7 @@ protected function setUp(): void
     {
         $this->cartMock = $this->createMock(Cart::class);
         $this->quoteMock = $this->getMockBuilder(Quote::class)
-            ->addMethods(['setIsMultiShipping', 'getIsMultiShipping'])
+            ->setMethods(['setIsMultiShipping', 'getIsMultiShipping'])
```
Comment (lbajsarowicz) [inline InventoryInStorePickupMultishipping/Test/Unit/Controller/Checkout/PluginTest.php]: `setMethods` is deprecated. You should use either `addMethods` or `onlyMethods`, but as long as it's not throwing warning, you don't need to care about this too much (however it's being marked as deprecated with annotation already)

Comment (sidolov) [reply InventoryInStorePickupMultishipping/Test/Unit/Controller/Checkout/PluginTest.php]: I see, just found that test is failing due to method absence

Comment (lbajsarowicz) [reply InventoryInStorePickupMultishipping/Test/Unit/Controller/Checkout/PluginTest.php]: So we need to replace `onlyMethods` too :-( I don't know why you want to keep the compatibility with PHPUnit 6, as this is PR to PHP 7.4 branch, that drops backward compatibility with 6.

Comment (sidolov) [reply InventoryInStorePickupMultishipping/Test/Unit/Controller/Checkout/PluginTest.php]: I agree with you. Reverted the changes

**Review by sidolov — APPROVED** [review]

**Pull request merged**